config/routes/project.rb

```diff
@@ -17,7 +17,13 @@
       get :edit
       post :transfer
       resources :members, only: %i[create destroy index new update]
-      resources :samples
+      resources :samples do
+        scope module: :samples do
+          collection do
+            resources :transfer, only: %i[create new]
```

Comment:
Change this `resources` to `resource` so that we don't get an index path

Reply:
Adding `:controller => :transfer` fixed the above pluralisation, thank you. Please see 22ecef81bdf9ffc5ec5dcc1af6b12b79f91b02bf.

Reply:
Please see 572f9fb59c3cbfb316661839c85d3b62d045087b for pluralised transfer controller routes.

Reply:
Thanks for updating this!